pypsa_to_IPCC.py

```diff
@@ -65,19 +67,11 @@
 
 for scenario in scenarios:
     #one excel file per scenario
-    file = openpyxl.load_workbook(path)
+    file = openpyxl.load_workbook(template_path)
     ds = file['data'] #data sheet
     for year in years:
-        if scenario == 'Early and Steady':
-            n=pypsa.Network('data/version-{}/postnetworks/postnetwork-go_TYNDP_{}.nc'.format(scenario_short_name[scenario], year))
-        elif scenario == 'Late and Rapid':
-            n=pypsa.Network('data/version-{}/postnetworks/postnetwork-wait_TYNDP_{}.nc'.format(scenario_short_name[scenario], year))
-        elif scenario == 'Transmission expansion after 2030':
-            n=pypsa.Network('data/version-{}/postnetworks/postnetwork-go_opt_{}.nc'.format(scenario_short_name[scenario], year))
-        else:
-            n=pypsa.Network('data/version-{}/postnetworks/postnetwork-go_TYNDP_{}.nc'.format(scenario_short_name[scenario], year))
-        
-        costs = pd.read_csv("data/costs/costs_{}.csv".format(year), index_col=[0,1])
+        n = pypsa.Network(f"{scenarios[scenario]}{year}.nc")
+        costs = pd.read_csv(f"costs/costs_{year}.csv", index_col=[0,1])
```

Comment:
this requires explaining in the README that not only the network file is necessary to create the outputs file but also the files "cost/costs_{year}.csv